src/Trakx.CoinGecko.ApiClient.Tests/TestInitializer.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using Trakx.Utils.Extensions;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+[assembly: Xunit.TestFramework("Trakx.CoinGecko.ApiClient.Tests.TestInitializer", "Trakx.CoinGecko.ApiClient.Tests")]
```

Comment:
nice, so that is something that gets loaded every time one runs the test suite, completely or partially?
👍

Reply:
This one is executed 1 time when test suite is executed.